src/main/java/uk/gov/justice/hmpps/prison/service/CaseNoteService.java

```diff
@@ -179,7 +179,7 @@ public CaseNote createCaseNote(String offenderNo, @NotNull @Valid @CaseNoteTypeS
     }
 
     @Transactional
-    @VerifyOffenderAccess(overrideRoles = {"SYSTEM_USER"})
```

Comment:
This is only called by prison-staff-hub and whereabouts-api which do not have SYSTEM_USER role, so therefore can be removed.